exercises/numbers/isEven/isEven.js

```diff
@@ -15,7 +15,12 @@
  */
 function isEven(num) {
   // Hint: See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder
-  return _____;
+  if (num % 2 === 0){
```

Comment:
Two things.

**First**, try to keep formatting / indentation consistent. It makes it easier for reviewers / teammates to follow and makes it easier to notice when something is off about your code.

   From the project root directory you can run `npm install` to install some libraries associated with the project. Included in that is something called `ESLint` which will highlight when your formatting is off.

   You can also press Command+Shift+P to open the VS Code command pallette and run "Format Document". (Ctrl+Shift+P on Windows).

**Second**, because `num % 2 === 0` evaluates directly to `true` or `false` you don't need the conditional.

If you do:

```js
function isEven(num) {
  return num % 2 === 0;
}
```

you will return whatever `num % 2 === 0` evaluates to (which will be either `true` or `false`).